spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe OpenLibraryService do
+  
+  
+  describe '#book_info' do
+    subject(:book_info) { OpenLibraryService.new.book_info(isbn) }
+
+    context 'When ISBN exists' do
+      let(:isbn) { '0385472579' }
```

Comment:
If you can try to use faker here :)

Reply:
I added faker here but I have to modify a lot of things in the way I stub the request 🤔